### ISSUE TITLE
Mapping of (more/all) mp3 ID3v2.x standard tags

### ISSUE THREAD

**Issue by frank1969b** [LMS-Community/slimserver#1156]
Today some (most) "Standard" mp3 ID3v2.x tags are mapped by LMS / Scanner, some are missing like
TMED = MEDIATYPE
TEXT = LYRICIST
so "View tabgs" show different field names for mp3-files than on FLAC, OGG etc.

Maybe we should add 
TMED
TEXT
to the "transformation" table in
https://github.com/LMS-Community/slimserver/blob/a42d7cc589de9e02fb66ac7197e429c6be5a46f5/Slim/Formats/MP3.pm#L69
or even all "standard" tags listed here:
https://docs.mp3tag.de/mapping/

Maybe we could also unifiy/standardize the way, "View tags" shows them
(today e.g. it shows ALBUM for FLAC/OGG... files but TALB for mp3 files.

Already discussed here:
https://forums.slimdevices.com/forum/developer-forums/beta/1724218-user-defined-contributor-roles-are-only-read-from-flac-files

**Comment by michaelherger** [comment]
"View Tags" is, as the name suggests, to view the tags as they are. This is a raw view by design, no mapping or other applied. It would even show you tags which aren't scanned (IIRC). That won't change.

But as for the tag mapping - I'd be happy to merge a pull request. It's not highly technical, and as you've already got two nice test cases I'm sure you'll figure out how to do it 😉.

**Comment by frank1969b** [comment]
OK, I tried and have a mp3.pm ready, but don't know how to get it to You - tried with "new pull request" but the system only tells me "there's nothing to compare"...

**Comment by michaelherger** [comment]
You'll have to fork the repository on Github, add your own fork as a "remote" to your local clone. Then you push the change to your fork, rather than to this repository directly. Once that's done you can go to your Github page and create the pull request.

Or you send me the file and I'll see what I can do 😁.

**Comment by michaelherger** [comment]
I was wondering whether this issue (or parts of it) have been covered by some of @darrell-k's recent work? Eg. the `LYRICIST` role?

**Comment by darrell-k** [comment]
Yes, you could import LYRICIST/TEXT or anything else as a user-defined role. Things that aren't contributors need a mapping to a database column.
